Dockerfile

```diff
@@ -2,9 +2,12 @@ FROM golang:latest as builder
 
 WORKDIR /go/src/github.com/micromdm/micromdm/
 
+ARG TARGETARCH
```

Comment:
Do these have reasonable defaults or are they mandatory when doing a docker build?

Reply:
There are [implicit default values](https://docs.docker.com/engine/reference/builder/) for this already, which are overridden by the `--platform` flag in the buildx step.

Example Dockerfile:
```
FROM alpine
ARG TARGETPLATFORM
RUN echo "$TARGETPLATFORM"
```

Running this on an x86 machine running macOS:
```
docker build .
...
 => [2/2] RUN echo "linux/amd64"
 ```
 vs
 ```
docker build --platform linux/arm .
 ...
=> [2/2] RUN echo "linux/arm/v7"
``